flipperdevices/flipperzero-firmware
# [FL-3846] Event Loop Timers

# What's new

- Add software timer functionality to `FuriEventLoop`
- Add a test blinky application
- Refactor the `Dolphin` application

# Verification 

1. Connect 6 LEDs to pins 2-7 on Flipper (use series resistors of course)
2. Flash the firmware first and then run the test application: `./fbt launch APPSRC=event_loop_blink_test`
3. Observe the blinkenlights! Each button on the Flipper can stop or start a particular LED. Press and hold `Back` to exit.
4. Speed up the Dolphin: compile with `--extra-define=DOLPHIN_DEBUG`, observe the messages in the log.

# Checklist (For Reviewer)
